time stamp
multisense_ros/SensorStatus status
float32 desired_rate
float32 current_rate
